run_lualatex.py

```diff
@@ -54,6 +54,7 @@
 env["TEXMFCNF"] = os.path.abspath("texmf/texmf-dist/web2c")
 env["TEXMFROOT"] = os.path.abspath("texmf")
 env["TTFONTS"] = ":".join(texinputs)
+env["max_print_line"] = "2147483647"
```

Comment:
Why is changing the `max_print_line` necessary for this package?

Reply:
Sorry it's not, this change comes from #62, I'll rebase that later today/tomorrow, it's not needed for this one.